front/src/routes/scene/edit-scene/actions/CalendarIsEventRunning.jsx

```diff
@@ -217,6 +217,9 @@ class CheckTime extends Component {
                 <option value="contains">
                   <Text id="editScene.triggersCard.calendarEventIsComing.contains" />
                 </option>
+                <option value="does-not-contain">
+                  <Text id="editScene.triggersCard.calendarEventIsComing.doesNotContain" />
+                </option>
```

Comment:
As this new mode is quite tricky to understand, I would add a small paragraph below it that's displayed only when the option is selected to explain exactly what's this mode is doing